ui/src/components/directory/topbar/eclSearch/builder/Refinement.vue

```diff
@@ -313,14 +323,31 @@ async function updateValueTreeRoots(): Promise<void> {
 async function updateIsValidProperty(): Promise<void> {
   if (props.focus?.iri === SNOMED.ANY) isValidProperty.value = true;
```

Comment:
use osQueryForPropertySearch for SNOMED.ANY to validate that the entity is a property and exists in the db/opensearch